verification/curator-service/ui/README.md

```diff
@@ -22,8 +22,24 @@ corresponding to this CSV format is available
 [here](https://docs.google.com/spreadsheets/d/1J-C7dq1rNNV8KdE1IZ-hUR6lsz7AdlvQhx6DWp36bjE).
 The following fields are currently incorporated:
 
+##### Case reference
```

Comment:
What about transmission, travel history, genome, other pathogens, and notes sections? Will those be added later?

Reply:
As above, can add later if we choose.